Package.swift

```diff
@@ -0,0 +1,37 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "mapbox-navigation-ios",
+    platforms: [.iOS(.v9)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "MapboxCoreNavigation",
+            targets: ["MapboxCoreNavigation"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(url: "https://github.com/mapbox/mapbox-accounts-ios/", .branch("1ec5-v2.3.1")),
+        .package(url: "https://github.com/mapbox/mapbox-directions-swift/", from: "1.2.0-alpha.3"),
+        .package(url: "https://github.com/mapbox/mapbox-events-ios/", from: "0.10.6"),
```

Comment:
Including this dependency seems to stall Xcode when resolving the dependency graph, eventually leading to the following error:

> the Package.resolved file is most likely severely out-of-date and is preventing correct resolution; delete the resolved file and try again